src/coprocessor/codec/error.rs

```diff
@@ -123,6 +135,20 @@ impl Error {
     pub fn unexpected_eof() -> Error {
         util::codec::Error::unexpected_eof().into()
     }
+
+    pub fn is_invalid_time_format(&self) -> bool {
+        self.code() == ERR_TRUNCATE_WRONG_VALUE
```

Comment:
I can't see the relationship with a `invalid_time_format` and `code == ERR_TRUNCATE_WRONG_VALUE`. Consider just removing this function?

Reply:
fixed